crates/chia-protocol/src/bytes.rs

```diff
@@ -44,6 +44,12 @@ impl Bytes {
     }
 }
 
+impl Bytes32 {
```

Comment:
I think it would be fine to change the main `BytesImpl` constructor to be `const` instead of this

Reply:
I initially tried this and had a problem with cargo not liking it due to the variable size of the array.

Reply:
in `BytesImpl`, `N` is not variable, it's a constant